lib/rbac/seed.rb

```diff
@@ -119,10 +125,8 @@ def find_uuid(type, data, name)
       result.uuid
     end
 
-    def create_request(user_file)
-      raise "File #{user_file} not found" unless File.exist?(user_file)
-      user = YAML.load_file(user_file)
-      {:headers => {'x-rh-identity' => Base64.strict_encode64(user.to_json)}, :original_url => '/'}
+    def create_request
```

Comment:
@syncrou We should be passing the x-rh-identity that we receive and not fabricate a new one. The pass_thru headers in the service should handle it.

Reply:
The thought here originally was to allow for the seed class to use a User file from the rake task or use the user from the request if that is sent in.